### ISSUE TITLE
implement WRITE_TIFF

### ISSUE THREAD

**Issue by slayoo** [gnudatalanguage/gdl#121]
As reported on SF.net (https://sourceforge.net/p/gnudatalanguage/feature-requests/19/):

```
Although ImageMagick seems to support 32-bit tiffs,
GDL:s read_tiff does not read them (magick_cl.pp
throws an exception if the bit depth is not 8 or 16).
It would be great if this capability could be added.

huldt@xray.bmc.uu.se
```

Further comment:

```
Assuming 32 bits refers to the number of bits PER CHANNEL.
I'm not sure if IDL supports this. I'll have to check if
adding the additional code causes a problem elsewhere in
magick_cl.cpp

CL.
```

**Comment by rexso** [comment]
#288 supports 32-bit tiff for **QUERY_TIFF** and **READ_TIFF**.
WRITE_TIFF is yet to be implemented using this *libtiff* version.

**Comment by slayoo** [comment]
The TIFF support is now in!
Renaming to indicate that only WRITE_TIFF is missing

**Comment by brandy125** [comment]
Are there any plans to implement the GEOTIFF keyword in write_tiff?

**Comment by GillesDuvert** [comment]
implemented in #1549
